Marlin/src/lcd/e3v2/proui/dwin.cpp

```diff
@@ -2555,7 +2559,7 @@ void setFlow() { setPIntOnClick(FLOW_EDIT_MIN, FLOW_EDIT_MAX, []{ planner.refres
 
   void manualMeshSave() {
     LCD_MESSAGE(MSG_UBL_STORAGE_MESH_MENU);
-    queue.inject(F("M211S1\nM500"));
+    queue.inject(F("M211S1"));
```

Comment:
This doesn't pertain to UBL. See line 2061.
```cpp
TERN_(MESH_BED_LEVELING, manualMeshSave());
```

Reply:
Also, `M211S1` enables software endstops. I'm not sure how that is pertinent here.